flixel/addons/effects/FlxGlitchSprite.hx

```diff
@@ -69,17 +86,12 @@ class FlxGlitchSprite extends FlxSprite
 		}
 		else
 		{
-			_time += elapsed;
+			_time += FlxG.elapsed;
```

Comment:
It's somwhat tricky... if you call `target.update(elapsed)`, that results in two update calls per frame if both sprites are added.